tests/integration/components/feed-item-test.js

```diff
@@ -9,7 +10,43 @@ test('it renders', function(assert) {
   // Set any properties with this.set('myProperty', 'value');
   // Handle any actions with this.on('myAction', function(val) { ... });
 
-  this.render(hbs`{{feed-item}}`);
+  let article = {
+    author: {
+      image: 'https://static.productionready.io/images/smiley-cyrus.jpg',
+      username: 'Alon Bukai'
+    },
+    createdAt: moment('1995-12-25'),
+    favoritesCount: 9000,
+    body: 'This is feed item content',
+    slug: 'feed-item-title',
+    title: 'Feed Item Title'
+  };
 
-  assert.equal(this.$().text().trim(), '');
+  this.set('article', article);
+  this.render(
```

Comment:
I don't understand what you mean. That the hbs` should be on the same line as the render?
We have Prettier and ESLint setup that should be taking care of all formatting and syntax issues.
Pretty much what they decide goes.